src/context.ts

```diff
@@ -28,7 +34,7 @@ class Context {
    * // Returns: {owner: 'username', repo: 'reponame', path: '.github/stale.yml'}
    *
    */
-  repo (object) {
+  repo<T> (object: T) {
```

Comment:
object is optional here, it can be undefined
replace with `repo<T> (object?: T)`